datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -2020,14 +2020,6 @@ statement ok
 drop table t;
 
 
-
-
-statement error DataFusion error: Execution error: Table 't_source' doesn't exist\.
```

Comment:
drive by cleanup -- this looks like it was a copy/paste of the statements immediately above, so I removed the duplication